tutorials/build_system/prerequisites.md

```diff
@@ -2,122 +2,201 @@
+  If needed, the IntelliJ IDEA Gradle plugin will download the version of Gradle specified in this file.
+* The shell scripts `gradlew` and `gradlew.bat` for executing the plugin build with the Wrapper files. 
+* The `settings.gradle` file, containing a definition of the `rootProject.name`.
+* The structure under the `src` directory is the default Gradle [`SourceSet`](https://docs.gradle.org/current/userguide/java_plugin.html#sec:java_project_layout), tailored for an IntelliJ Platform Plugin.
+
+```text
+  my_gradle_plugin/
+    build.gradle
+    gradle/
+      wrapper/
+        gradle-wrapper.jar
+        gradle-wrapper.properties
+    gradlew
+    gradlew.bat
+    settings.gradle
+    src/
+      main
+        java/
+        resources/
+          META-INF/
+            plugin.xml
+      test
+        java/
+        resources/
+```
+
+The New Project Wizard produces the `build.gradle` file shown below:
+* There is no explicit `buildscript{}` in the file.
+  The IntelliJ IDEA Gradle plugin dynamically creates a `buildscript{}`.
+* Two plugins to Gradle are explicitly declared:
+  * The [Gradle Java](https://docs.gradle.org/current/userguide/java_plugin.html) plugin.
+  * The [IntelliJ IDEA Gradle plugin](https://github.com/JetBrains/gradle-intellij-plugin/).
+* The _Group ID_ from the Wizard [Project Naming Screen](#project-naming-screen) is the `(project.)group` value.
+* The _Version_ from the Wizard [Project Naming Screen](#project-naming-screen) is the `(project.)version` value.
+* The `sourceCompatibility` line is injected to enforce using JDK 1.8.
+* The only comment in the file is a link to the `README.md` for the IntelliJ IDEA Gradle plugin, which is a reference for the DSLs defined by the plugin.
+* The value of the Setup DSL attribute `intellij.version` specifies the version of the IntelliJ Platform to be used to build the plugin.
+  It defaults to the version of IntelliJ IDEA that was used to run the Wizard.
+* The value of the Patching DSL attribute `patchPluginXml.changeNotes` is set to place holder text.
+
+```text
+  plugins {
+      id 'java'
+      id 'org.jetbrains.intellij' version '0.4.8'
+  }
+  
+  group 'com.your.company'
+  version '1.0' 
+  sourceCompatibility = 1.8
+  
+  repositories {
+      mavenCentral()
+  } 
+  dependencies {
+      testCompile group: 'junit', name: 'junit', version: '4.12'
+  }
+  
+  // See https://github.com/JetBrains/gradle-intellij-plugin/
+  intellij {
+      version '2019.1'
+  }
+  patchPluginXml {
+      changeNotes """
+        Add change notes here.<br>
+        <em>most HTML tags may be used</em>"""
+  }
+```
 
-> **Note** Please make sure to always upgrade to latest version of `gradle-intellij-plugin` plugin, follow releases on [GitHub](https://github.com/JetBrains/gradle-intellij-plugin/releases). 
+### Opening a Wizard-Generated Gradle Project for the First Time
+The first time a project is opened an _Import Project from Gradle_ dialog appears.
```

Comment:
'Import from Gradle' shouldn't appear if you open the project created as described here. Please make sure that you select the project directory in 'Open Project' chooser, not build.gradle file.

Reply:
Thank you, I will update the description.